setup.py

```diff
@@ -1,9 +1,69 @@
 import setuptools
 import os
 import subprocess
+import platform
 from os import path
 import sys
 
+def is_32bit():
+    return platform.architecture()[0] == '32bit'
```

Comment:
`is_64bits = sys.maxsize > 2**32`

Check out the NOTE at this link:
https://docs.python.org/3/library/platform.html#cross-platform